Apps/Inc/Voltage.h

```diff
@@ -14,6 +14,11 @@
 #include "common.h"
 #include "os.h"
 
+typedef struct system_danger{
+    SafetyStatus wire_checks[TOTAL_VOLT_WIRES];
+    SafetyStatus module_checks[NUM_BATTERY_MODULES];
+} Safety_Info;
```

Comment:
I would make the struct more specific to Voltage.c since Safety_Info could be applied to Temperature.c and Amperes.c. Maybe VoltageSafetyChecks or something that has "voltage" in the same